src/core/db/repositories/peopleRepository.ts

```diff
@@ -25,11 +25,45 @@ export function getAllPeople(): PersonEntity[] {
   return people;
 }
 
-export function createPersonInDb(name: string, active: boolean): any {
+export function getPersonByID(PersonId: number): PersonEntity {
   const people = readPeopleFromFile();
-  console.log(JSON.stringify(people));
-  people.push(new PersonEntity(10, name, active, Date.now()));
+  const person =
+    people.find(({ id }) => id == PersonId) ||
+    new PersonEntity(11, 'Nan', false, Date.now());
```

Comment:
If you want to handle case where the item is not found, then it should return `Not found` error instead of some random item.